test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/conftest.py

```diff
@@ -56,6 +56,7 @@
 
 def pytest_addoption(parser):
     parser.addoption('--region', default='us-west-2')
+    parser.addoption('--framework-version', default='')
```

Comment:
The `versions` input is meant to handle incoming framework versions.

Reply:
I was having a test failure for not having this field back in Jan, will remove it from this PR.